src/register-org/components/payment-by-account/payment-by-account.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, ElementRef, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { ErrorMessage } from '../../../shared/models/error-message.model';
+import { RegisterComponent } from '../../containers/register/register-org.component';
+import { RegisterOrgService } from '../../services/register-org.service';
+
+@Component({
+  selector: 'app-payment-by-account',
+  templateUrl: './payment-by-account.component.html'
+})
+export class PaymentByAccountComponent extends RegisterComponent implements OnInit, OnDestroy {
+  @ViewChild('errorSummaryTitleElement') public errorSummaryTitleElement: ElementRef;
+
+  public pbaFormGroup: FormGroup;
+  public pbaError: ErrorMessage;
+
+  constructor(public readonly router: Router,
+    public readonly registerOrgService: RegisterOrgService
```

Comment:
@johnbenjamin-hmcts - same thing as above, on use of `public`

Reply:
@Daniel-Lam - All the components extend from the master page component called `RegisterComponent` that takes in Router and RegisterOrgService as the constructor properties.

Having them as private on both the master page component and child components did not work.